runner_matchmaker_regex.py

```diff
@@ -6,45 +6,37 @@
 def regex_string_maker(col_nums):
     base_string = "([A-Z]{3,} [A-Z]{3,}|[A-Z]{3,})(  .*\\n)"
     end_string = ""
+
     for i in range(col_nums):
         end_string += base_string
+
     return end_string
 
 
 def to_dict(matches):
     mushroom_list = []
+
     for match in matches:
-        #Key = even
-        #Value = odd
-        match_list_even = match[0::2] #Even GROUPS
-        match_list_odd = match[1::2]  #Odd GROUPS
+        # Key = even
+        # Value = odd
+        match_list_even = match[0::2]  # Even GROUPS
+        match_list_odd = match[1::2]  # Odd GROUPS
         output = dict(zip(match_list_even, match_list_odd))
         dict_to_sql(output, session)
-        #mushroom_list.append(output)
-    return output
+        mushroom_list.append(output)
 
+    return mushroom_list
```

Comment:
You were using `output` here but the vairable is only defined inside of the `for in` loop.

If you wanted to return output, you have to have return in the same block as line 25 or define `output` outside of the `for in` loop.